File: taller-1/python/taller1-tienda.py
producto = input("Ingrese el nombre del producto: ")
precio = float(input("Ingrese el precio por unidad: "))
cantidad= int(input("Ingrese la cantidad de unidades: "))

total = precio * cantidad
descuento = 0.15 if cantidad > 10 else 0.10 if cantidad > 5 else 0.05
total_con_descuento = total - (total*descuento)

print(f"Resumen de compra:\n"
      f"Producto: {producto}\n"
      f"Precio total sin Descuento: ${total:.2f}\n"
      f"Descuento aplicado: {descuento*100}%\n"
      f"Precio Final: ${total_con_descuento:.2f}"
      
      )

File: taller-1/js/taller1-tienda.js
const prompt = require('prompt-sync')();

let producto = prompt ("Ingrese el nombre del producto: ");
let precio = parseFloat(prompt("Ingrese el precio por unidad: "));
let cantidad = parseInt(prompt("Ingrese la cantidad de unidades: "));

let total = precio * cantidad;
let descuento = cantidad >10 ? 0.15: cantidad>5?0.10 : 0.05;
let totalConDescuento = total - (total*descuento)


console.log(`Resumen de compra:
Producto: ${producto}
Precio total sin descuento: ${total.toFixed(2)}
Descuento aplicado: ${descuento}
Precio Final ${totalConDescuento.toFixed(2)}
`)